static/app/views/onboarding/setupDocs.tsx

```diff
@@ -68,81 +68,160 @@ function MissingExampleWarning({
   );
 }
 
+function LoadGettingStartedDoc({
+  platform,
+  orgSlug,
+  projectSlug,
+  activeProductSelection,
+}: {
+  activeProductSelection: PRODUCT[];
+  orgSlug: Organization['slug'];
+  platform: PlatformIntegration | null;
+  projectSlug: Project['slug'];
+}) {
+  const [module, setModule] = useState<null | {
+    default: React.ComponentType<{activeProductSelection: PRODUCT[]; dsn: string}>;
+  }>(null);
+
+  const platformPath =
+    platform?.type === 'framework'
+      ? platform?.id.replace(`${platform.language}-`, `${platform.language}/`)
+      : platform?.id;
+
+  const {
+    data: projectKeys = [],
+    isError: projectKeysIsError,
+    isLoading: projectKeysIsLoading,
+  } = useApiQuery<ProjectKey[]>([`/projects/${orgSlug}/${projectSlug}/keys/`], {
+    staleTime: Infinity,
+  });
+
+  useEffect(() => {
+    if (projectKeysIsError || projectKeysIsLoading) {
+      return;
+    }
+
+    async function getGettingStartedDoc() {
+      const mod = await import(`sentry/gettingStartedDocs/${platformPath}`);
+      setModule(mod);
+    }
+    getGettingStartedDoc();
+  }, [platformPath, projectKeysIsError, projectKeysIsLoading, projectKeys]);
+
+  if (!module) {
+    return null;
+  }
+
+  const {default: GettingStartedDoc} = module;
+  return (
+    <GettingStartedDoc
+      dsn={projectKeys[0].dsn.public}
```

Comment:
does anyone know why the `projectKeys` is an array? Didn't know a project could have multiple DSNs 🤔

Reply:
You can generate as many as you want yes

Reply:
that is interesting. So if we have multiple keys do we display all of them in the settings? In any case this sounds like irrelevant for the project creation and onboarding flow